src/components/common/YourCompany/ResolutionDate.vue

```diff
@@ -0,0 +1,229 @@
+<template>
+  <section id="resolution-date">
+    <v-row no-gutters>
+      <v-col cols="3" class="pr-2">
+        <label :class="{'error-text': invalidSection}">
+          <strong>Resolution Date</strong>
+        </label>
+        <v-chip
+          v-if="isCorrected"
+          x-small label
+          color="primary"
+          text-color="white"
+        >
+          <span>{{ editedLabel }}</span>
+        </v-chip>
+      </v-col>
+
+      <!-- Info tooltip-->
+      <v-col cols="9">
+        <span v-if="!onEditMode" class="info-text mr-1">{{ resolutionDate }}</span>
+        <v-tooltip
+          top
+          content-class="top-tooltip"
+          transition="fade-transition"
+          nudge-right="3"
+        >
+          <template v-slot:activator="{ on }">
+            <v-icon v-on="on" class="info-icon">mdi-information-outline</v-icon>
+          </template>
+          <span>If the resolution date is incorrect, it must be corrected through a correction filing.</span>
+        </v-tooltip>
+
+        <!-- Edit mode -->
+        <template v-if="onEditMode">
+          <div cols="12" sm="3" class="pr-4">
+            <label class="start-date-title title-label">Special Resolution Date</label>
+            <p class="mt-4">
+              Enter the start date of the business. The start date can be
+              <v-tooltip top max-width="20rem" content-class="top-tooltip" transition="fade-transition">
+                <template v-slot:activator="{ on }">
+                  <span v-on="on" class="tool-tip dotted-underline">no more than 2 years in the past</span>
+                </template>
+                <span>Choose the oldest date possible even if the actual start date is older than 2 years in the
+                  past.</span>
+              </v-tooltip>
+              and 90 days in the future. Make certain that this is the correct date as it cannot be easily
+              corrected afterwards.
+            </p>
+            <DatePickerShared
+              title="Resolution Date"
+              nudge-right="80"
+              nudge-top="15"
+              :minDate="startDateMinStr"
+              :maxDate="startDateMaxStr"
+              @emitDate="onOkClicked($event)"
+            />
+          </div>
+          <div class="float-right mb-2">
+            <v-btn large color="primary" id="start-done-btn" class="mr-2" @click="onDoneClicked()">
+              <span>Done</span>
+            </v-btn>
+            <v-btn large outlined color="primary" id="start-cancel-btn" @click="onCancelClicked()">
+              <span>Cancel</span>
+            </v-btn>
+          </div>
+        </template>
+
+        <span class="mt-n2 mr-n3 float-right">
+          <!-- Correct changes button -->
+          <v-btn v-if="isCorrectionFiling && !onEditMode && !isCorrected" text color="primary"
```

Comment:
Combine these conditions in a getter maybe?

`isCorrectionFiling && !onEditMode `
you are repeating it 3 times